Helper/Product.php

```diff
@@ -46,6 +46,20 @@ public function getProductData(MagentoProduct $product)
         }
         return $data;
     }
+    
+    public function getProductFullData($parentProduct, $product = null)
```

Comment:
it's a bit confusing now: what is `$product` here? I assume it's a variant product. If yes, I suggest to rename it to `$variantProduct = null`